doc/source/examples/incompressible-flow/3d-flow-around-sphere/flow-around-sphere.rst

```diff
@@ -0,0 +1,329 @@
+    set max refinement level    = 3
+    set variable		            = pressure
+  end
+
+The mesh is dynamically adapted based on an estimate of the error of the solution for the pressure (the Kelly error estimator). The refinement is based on the number of elements. This means that the number of cells refined/coarsened per iteration is based on the fraction of the number of cells, rather than the fraction of the error (where all cells which have the fraction of the error are refined/coarsened).
+
+The ``min refinement level`` refers to the base mesh which has been used in the previous static simulations. The mesh can only become finer than this, not coarser. The ``max refinement level`` is set at 3, giving a maximum possible number of cells of 3 million. However, the ``max number elements`` limits the number of cells to 50,000 to keep the simulation within feasible computational expense.
+
+
+Running the simulation
+----------------------
+Launching the simulation is as simple as specifying the executable name and the parameter file. Assuming that the ``gls_navier_stokes_3d`` executable is within your path, the simulation can be launched by typing:
+
+.. code-block:: text
+
+  gls_navier_stokes_3d sphere_0.1.prm
+
+or 
+
+.. code-block:: text
+
+  gls_navier_stokes_3d sphere_150.prm
+
+or
+
+.. code-block:: text
+
+  gls_navier_stokes_3d sphere_adapt.prm
+
+Lethe will generate a number of files. The most important one bears the extension ``.pvd``. It can be read by popular visualization programs such as `Paraview <https://www.paraview.org/>`_. 
+
+
+First case results (Re = 0.1)
+-----------------------------
+
+Using Paraview, the steady-state velocity profile and the pressure profile can be visualized by operating a *slice* along the xy-plane (z-normal) that cuts in the middle of the sphere (See documentation <https://forgeanalytics.io/blog/creating-slices-in-paraview/>)_.
+
+.. image:: images/velocity_0.1.png
+    :alt: velocity distribution 0.1
+    :align: center
+
+.. image:: images/pressure_0.1.png
+    :alt: pressure distribution 0.1
+    :align: center
+
+We can appreciate the axisymmetrical behavior of the flow. The drag on the sphere is available in the output file ``force.00.dat`` (the other force files ``force.01.dat`` and ``force.02.dat`` give the forces on boundary conditions 1 and 2 respectively). 
+
+.. note::
+  The last line of the file shows the force calculated in the last iteration. Since the flow in the x-direction, the x-direction force ``f_x`` gives the drag force.
+
+.. code-block:: text
+
+  cells      f_x           f_y          f_z      
+  5823 98.3705224612 -0.0000000785 0.0000001119
+
+Given the flow parameters, the calculated drag coefficient is 250.50, using 6000 cells. At Re=0.1, an analytical solution of the drag coefficient is known: :math:`C_D = 240`. The deviation from the analytical solution is primarily due to the coarseness of the mesh. It would be relevant to carry out a mesh refinement analysis.
```

Comment:
The deviation is much more likely to come from the size of the domain for a very low Reynolds number.

Reply:
I will add this :)